go.mod

```diff
@@ -22,16 +23,14 @@ require (
 )
 
 require (
-	github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412 // indirect
+	// We must use the same version as the Terraform provider.
+	github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7
```

Comment:
**To the reviewer:**

The goal is to move away from the old unmaintained fork of the **openpgp** package to the ProtonMail's one. That said, we do not strictly _necessarily_ need to match the same version that the Terraform AWS provider is using - this is only done primarily for compatibility reasons.